Framework/MDAlgorithms/test/ConvertToMDTest.h

```diff
@@ -89,7 +89,7 @@ class ConvertToMDTest : public CxxTest::TestSuite {
     TS_ASSERT_THROWS_NOTHING(pAlg->initialize())
```

Comment:
It would be really nice to get some reviewable (possibly side-by-side) performance checks for this too. Based on what you have already done in the system tests, I think you could easily port something into `ConvertToMDTestPerformance` [here](https://github.com/mantidproject/mantid/blob/master/Framework/MDAlgorithms/test/ConvertToMDTest.h#L565)
These get reviewed weekly by Dan Nixon for regressions. Example [here](http://builds.mantidproject.org/job/master_performancetests2/Master_20branch_20performance_20tests/)